crates/context_aware_config/src/api/dimension/types.rs

```diff
@@ -4,7 +4,7 @@ use serde_json::Value;
 #[derive(Debug, Deserialize)]
 pub struct CreateReq {
     pub dimension: String,
-    pub priority: u16,
+    pub priority: i32,
```

Comment:
If we are expecting only positive numbers, maybe use u32?
```suggestion
    pub priority: u32,
```

Reply:
avoided this, as we needed i32 in the next ops,
and then conversion from u32 to i32 is more likely to panic

Reply:
Where exactly @ayushjain17 ?